cfgov/jinja2/v1/ask-cfpb/answer-search-results.html

```diff
@@ -44,7 +44,7 @@ <h3 class="results-header">Results for ‘{{ page.query }}’</h3>
                 <div class="question_list">
                 {% for question in results %}
                     <article class="question_summary question_summary__full">
-                        <p class="question_title"><a href="{{ question[0] }}" class="kbq">{{ question[1]|safe }}</a></p>
+                        <p class="question_title"><a href="{{ question[0] }}/" class="kbq">{{ question[1]|safe }}</a></p>
```

Comment:
This one isn't needed, because Haystack search results deliver a URL with a trailing slash, so we'll end up with double end slashes.
E.g., result URLs at https://www.consumerfinance.gov/ask-cfpb/search/?q=tuition already end with slashes.

Reply:
ah-- cool.

Reply:
fixed!